cmd/infra/powervs/destroy.go

```diff
@@ -206,6 +222,50 @@ func (options *DestroyInfraOptions) DestroyInfra(infra *Infra) (err error) {
 	return
 }
 
+// deleteDNSRecords deletes DNS records from CIS domain
+func deleteDNSRecords(options *DestroyInfraOptions) error {
+
+	if options.CisCRN == "" || options.CisDomainID == "" {
+		if options.BaseDomain == "" || options.Name == "" {
```

Comment:
is there reason why we made these 2 options as optional? if we make them mandatory then we don't need this check!

Reply:
Yes right Manju, added them as mandatory and removed this check.